DataFormats/StdDictionaries/src/classes_others.h

```diff
@@ -28,10 +28,6 @@ namespace DataFormats_StdDictionaries {
   std::random_access_iterator_tag randaccitertag;
   std::set<int> dummy19;
   std::set<std::basic_string<char> > dummy20;
-  __gnu_cxx::new_allocator<char> dummy21;
```

Comment:
It was related to missing ROOT dictionaries, ROOT5 stuff. There should be  an issue in cmssw repository about that.